src/pages/TopupPage/TopupPage.tsx

```
@@ -0,0 +1,109 @@
+        {brand.variants.map((variant) => (
+          <div
+            style={{
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "start",
+              justifyContent: "center",
+              backgroundColor: theme.palette.background.paper,
+              padding: "10px 15px",
+              borderRadius: 10,
+            }}
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)